src/sdl2_compat.c

```diff
@@ -281,7 +281,7 @@ static char loaderror[256];
     }
 #elif defined(__unix__)
     #include <dlfcn.h>
-    #define SDL3_LIBNAME "libSDL3.so.0"
+    #define SDL3_LIBNAME "/home/franz/Documents/sdl-git/SDL/cmake-build-debug/libSDL3.so.0"
```

Comment:
You probably don't want to commit this. :)

Reply:
I'm looking into adding rpaths during build time (not install time) such that changes like these are not required anymore.

Reply:
Whoops, no I did not :)